extensions/Worker.Extensions.Http/src/DefaultFromBodyConversionFeature.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Serialization;
+using Microsoft.Azure.Functions.Worker.Extensions.Http.Converters;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Http
+{
+    internal class DefaultFromBodyConversionFeature : IFromBodyConversionFeature
```

Comment:
Are we expecting a matching implementation for the AspNetCore integration?

Reply:
Yes. I'll follow up with that in a separate PR soon.